### PR TITLE
🔨 GitHub Actions

### PR DESCRIPTION
**Pull request by cedricengagor**
## PR Details

This will allow us to replace our concourse pipeline with a free alternative called GitHub Actions. Using this system we don't need to leverage our own, which is beneficial for repos like this.

## How to test?

Requirements:
- https://www.docker.com/products/docker-desktop

Install:
- `brew install nektos/tap/act`
- `docker pull buildpack-deps`

Run:
- `act` in CLI inside `unified-components-react` root folder for all actions
- `act {name}` in CLI inside `unified-components-react` root folder for a specific one

### CODE REVIEW

**Review by matthias-deschoenmacker — COMMENTED** [review]
LGTM, but I am not an expert on this. Can I test this out?

**Comment by cedricengagor** [comment]
> LGTM, but I am not an expert on this. Can I test this out?

Added the how to test for ya

**Review by matthias-deschoenmacker — APPROVED** [review]
LGTM, didn't test it tho

File: .github/workflows/code-styles.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: '16.x'
+          cache: 'npm'
```
Comment (cedricengagor) [inline .github/workflows/code-styles.yml:47]:
```suggestion
```

File: .github/workflows/code-styles.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
```
Comment (cedricengagor) [inline .github/workflows/code-styles.yml:73]:
```suggestion
```

File: .github/workflows/storybook.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: '16.x'
+          cache: 'npm'
```
Comment (cedricengagor) [inline .github/workflows/storybook.yml:18]:
```suggestion
```

**Pull request merged**